src/cli/run_impl.rs

```diff
@@ -111,10 +89,6 @@ pub fn run(setup: Setup) {
 
     rpc_server.close();
     info!(target: "main", "Jsonrpc shutdown");
-
-    // FIXME: should gracefully shutdown network
-    network.close();
-    info!(target: "main", "Network shutdown");
```

Comment:
Why remove this?

Reply:
Moved to `network.rs`, network will shutdown when `NetworkController` dropped, just like how `ChainController` works.